test/unit/org/apache/cassandra/db/RecoveryManagerTruncateTest.java

```diff
@@ -106,7 +107,8 @@ public void testTruncate() throws IOException
 
         // and now truncate it
         cfs.truncateBlocking();
-        assert 0 != CommitLog.instance.resetUnsafe(false);
+        Map<Keyspace, Integer> replayed = CommitLog.instance.resetUnsafe(false);
+        assertFalse(replayed.isEmpty());
```

Comment:
`assertFalse` alone is not very helpful when debugging. what about logging `replayed` on assertion failure or sth along:
`assertThat(replayed, is(empty()));` (hamcrest)